### DISCUSSION TITLE
Update Lightshow and boombox similar to the music folder?

### DISCUSSION THREAD

**Discussion by prussell69** [marcone/teslausb#832] [Q&A]
I have setup both the Lightshow and boombox partitions on my Teslausb, and would like Teslausb to automatically update those partitions the same way the music partition is done. If new Lightshows are added on my server folder, they would be transfered (or removed) when the Teslausb archives.
I tried adding this to the configuration file:

export MUSIC_SHARE_NAME=Tesla/Music
export LIGHTSHOW_SHARE_NAME=Tesla/Lightshow
export BOOMBOX_SHARE_NAME=Tesla/Boombox

The music does just fine, but the others don't. Is it possible to have that feature added?
There is also the Locksound file that resides in one of the root directories of the partitions. That would also need to be transferred.

**Comment by VirtualJTW** [comment]
That would be very cool - I second this!

**Comment by stauffenberg2020** [comment]
I would very much love this too.

However I have another question. Why to emulate 4 USB devices for the car instead of 1. Why can't we have top level folders in emulated USB, say TeslaCam, Music, Lightshow, Boombox to do all of it one go. By this way, if we enable SAMBA drive, we can access all of the content in one go.

**Comment by marcone** [comment]
> Why can't we have top level folders in emulated USB, say TeslaCam, Music, Lightshow, Boombox to do all of it one go

Because [Tesla's documentation for Lightshow](https://github.com/teslamotors/light-show?tab=readme-ov-file#usb-flash-drive-requirements) says that  Lightshow drive must not contain a TeslaCam folder, and the [documentation for Boombox](https://www.tesla.com/ownersmanual/model3/en_us/GUID-79A49D40-A028-435B-A7F6-8E48846AB9E9.html#GUID-F5314683-2106-4326-A7FC-8C0DFB222139) similarly says the Boombox drive cannot be shared with Dashcam.
